### ISSUE TITLE
Improve the color contrast for some items

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/isotopes-and-atomic-mass#137]
**Test device**
Chromebook

**Operating System**
ChromeOS

**Browser**
Chrome

**Problem description**
From https://github.com/orgs/phetsims/projects/91/views/1?pane=issue&itemId=176924212&issue=phetsims%7Cqa%7C1388

Color contrast testing for the erase button passes, but on a Chromebook it appears noticeably lighter than expected. Even when you place the cursor over the button, it looks much lighter.

When it comes to the atoms in the buckets, the purple and green ones fail the color contrast test. This issue occurs with the ones that stand out on the surface of the bucket, especially those visible against a white background.

**Visuals**
<img width="141" height="139" alt="Image" src="https://github.com/user-attachments/assets/07a8b236-7e11-4835-91e6-1d1420713578" />

<img width="176" height="102" alt="Image" src="https://github.com/user-attachments/assets/bf513d6a-303e-4da4-b3dd-5dc23cf341f6" />

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/a7c31a01-b924-4513-a73f-5555b64f2a85" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Isotopes and Atomic Mass‬
URL: https://phet-dev.colorado.edu/html/isotopes-and-atomic-mass/1.2.0-rc.1/phet/isotopes-and-atomic-mass_all_phet.html
Version: 1.2.0-rc.1 2026-04-16 21:48:25 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (X11; CrOS x86_64 14541.0.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/146.0.0.0 Safari/537.36
Language: en-US
Window: 647x561
Pixel Ratio: 1.0507692098617554/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16383 imageUnits: 16 (vertex: 16, combined: 48)
Max viewport: 16383x16383
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by Nancy-Salpepi** [comment]
There are also issues with the text on the buckets, the white text doesn't have enough contrast with the lighter side of the bucket. 

<img width="225" height="126" alt="Image" src="https://github.com/user-attachments/assets/6705f71e-ef18-4a42-8735-9f17cf1ece6b" />

<img width="225" height="84" alt="Image" src="https://github.com/user-attachments/assets/b8d07286-ba52-4fa8-9e54-a4619511abcc" />

**Comment by Nancy-Salpepi** [comment]
<img width="892" height="413" alt="Image" src="https://github.com/user-attachments/assets/97e2d033-3fc7-4d44-bc13-e9eab99130dc" />

Based on the above, nothing needs to be done for the eraser and the atoms in the buckets. However, if easy @jbphet, it would be nice to add a thin black stroke around the individual atoms.

As far as the the text goes, how hard would it be to darken all the buckets and reduce the gradient--similar to what you did in BAA?

**Comment by jbphet** [comment]
I've added a stroke around the atoms and have darkened the gradient used on the buckets to increase contrast.  @Nancy-Salpepi and I reviewed, tested, and tweaked over Zoom, but it would still be good for her to double check this on main.  Assigning to her for review.  Please assign back to me after that so that I can cherry-pick to the RC branch.

**Comment by Nancy-Salpepi** [comment]
Changes look nice on main. Thanks @jbphet!

**Comment by vperezcadavid99** [comment]
It looks good for this rc.2 spot check. Thanks.
Closing!
